package-lock.json

```diff
@@ -0,0 +1,6947 @@
+{
```

Comment:
Woof, this is a big file to add to the diff. It looks purely additive, though. Is this a new requirement in newer version of Node, or was this an artifact of the build process?

Reply:
Yeah it's a requirement in newer versions of node, and they recommend you commit it to the repository.